docker-compose.link.yaml

```diff
@@ -13,7 +13,16 @@ services:
         npm run start:watch"
     networks:
       - ffc-demo
```

Comment:
If making the switch to use the default network (see comment on `docker-compose.yaml`), we'll need to add `- default` to the networks list here. This is because docker-compose only joins each service to its default networks if no other networks are listed for it. This would still reduce the amount of network configuration across all the docker-compose files to just this one line.

Reply:
I've made that change